src/rsdiv/evaluation/ranking_metrics.py

```diff
@@ -0,0 +1,56 @@
+from typing import Callable, Hashable, Sequence, TypeVar
+
+import numpy as np
+
+ItemNameType = TypeVar("ItemNameType", bound=Hashable)
+
+
+class RankingMetrics:
```

Comment:
Implement the `pytest` files, say `test_nDCg.py` and `test_map.py` under `test/evaluation` and pass the checks.